packages/excalidraw/element/newElement.ts

```diff
@@ -343,6 +347,9 @@ export const refreshTextDimensions = (
   if (textElement.isDeleted) {
     return;
   }
+  if (textElement.autoResize === false) {
+    return;
+  }
```

Comment:
I'm not sure about this ad-hoc disabling of this helper based on whether an element has `autoResize` disabled. Adds complexity / surprising behavior. E.g., if you expect something to refresh dimensions of an element, it should at most just not refresh the constrained axis, not bail completely.